joomla/joomla-cms
# [4.0] [backend-template] Custom select missing icon on focus

Pull Request for Issue # 27543

### Summary of Changes
extra class was present


### Testing Instructions
> ### Steps to reproduce the issue
> 1. Go to `/administrator/index.php?option=com_postinstall`
> 2. Look at the `Showing messages for` dropdown
> 3. Click it
> 
> You'll notice the arrow icon disappears
> 
> ![screeny3](https://user-images.githubusercontent.com/2019801/72441217-2f89a180-37a2-11ea-98dc-bbc15bb58b67.png)

### Expected result
Icon stays


### Actual result

icon vanishes on focus

### Documentation Changes Required

none
